prost-derive/src/field/scalar.rs

```diff
@@ -1,5 +1,5 @@
+use core::fmt;
 use std::convert::TryFrom;
```

Comment:
is this change necessary?  I wasn't under the impression that `prost-derive`'s _implementation_ would not be able to use stdlib APIs.

Reply:
it might not be, sorry